service/src/geopm_plugin.cpp

```diff
@@ -48,10 +50,17 @@ namespace geopm
 {
     void plugin_load(const std::string &plugin_prefix)
     {
+        std::string env_plugin_path_str(geopm::get_env("GEOPM_PLUGIN_PATH"));
         std::vector<std::string> plugin_paths {GEOPM_DEFAULT_PLUGIN_PATH};
         std::string so_suffix = ".so." GEOPM_ABI_VERSION;
         std::replace(so_suffix.begin(), so_suffix.end(), ':', '.');
 
+        if (!env_plugin_path_str.empty()) {
```

Comment:
This looks correct, but I'm surprised we didn't already have this feature...

Reply:
You had removed to back in April to get the build working.  It's in dev.